src/_guides/language/effective-dart/usage.md

```diff
@@ -441,11 +441,41 @@ var reciprocals = stuff.map((n) => 1 / n).cast<double>();
 {% endprettify %}
 
 
+## AVOID using `cast()` or `retype()`.
+
+This is the softer generalization of the previous rule. Sometimes there is no
+nearby operation you can use to fix the type of some object. Even then, when
+possible avoid using `cast()` or `retype()` to "change" a collection's type.
+
+Other options that are usually better are:
+
+*   Change the code where the collection is first created so that it has the
+    right type.
+
+*   If you are immediately iterating over the collection, cast each element
```

Comment:
you are immediately iterating -> you immediately iterate

Reply:
Done.